frontend/packages/console-shared/src/components/dashboard/top-consumers-card/ConsumersFilter.tsx

```diff
@@ -17,15 +17,15 @@ const toNanoSeconds = (value: React.ReactText) => {
 export const metricTypeMap: MetricTypeMap = {
   [MetricType.CPU]: {
     description: CPU_DESC,
-    humanize: (value) => humanizeSeconds(toNanoSeconds(value)),
+    humanize: (value) => humanizeSeconds(toNanoSeconds(value as React.ReactText)),
```

Comment:
Why is `as` needed here?

Reply:
`toNanoSeconds` expects `React.ReactText` however the data type of `value` is expanded to accept `DataPoints[]`